config/pmd-test.xml

```diff
@@ -99,28 +96,33 @@
            In MainTest PMD does not find asserts in lambdas called in the method
              invokeAndWait. -->
       <property name="violationSuppressXPath"
-               value="//ClassOrInterfaceDeclaration[@SimpleName='AllChecksTest'
+               value="//ClassDeclaration[@SimpleName='AllChecksTest'
                      or @SimpleName='AstRegressionTest'
-                     or @SimpleName='ImportControlCheckTest']//PrimaryPrefix/Name[@Image='verify']
-                   | //ClassOrInterfaceDeclaration[@SimpleName='MainTest']
+                     or @SimpleName='ImportControlCheckTest']//PrimaryPrefix/Name[@Name='verify']
+                   | //ClassDeclaration[@SimpleName='AllChecksTest']
+                       //MethodDeclaration[@Name='testAllModulesAreReferencedInConfigFile'
```

Comment:
please mark all such extra violations with comment "until #xxxxxxx " (new issue your are created) to find such items easily and clearl know they are temporal, not a permanent